recipe/meta.yaml

```diff
@@ -10,37 +10,29 @@ source:
   sha256: 702f1dffc5799e47d879383bbf66bf4f1baa92e40e6241dc9e59845b6ff4df2c
 
 build:
-  number: 1
+  number: 2
   script:
     - "{{ PYTHON }} -m pip install . --no-deps -vv"
 
 requirements:
   host:
-    - future
+    # - future
     - python
     - pip
-    - numpy ==1.11.3 py27hab51878_1207  # [win and py2k]
-    - numpy  # [not (win and py2k)]
-    - scipy ==1.2.1 py27h4c3ab11_0  # [win and py2k]
-    - scipy  # [not (win and py2k)]
-    - libblas ==3.8.0 8_mkl  # [win and py2k]
-    - libcblas ==3.8.0 8_mkl  # [win and py2k]
-    - liblapack ==3.8.0 8_mkl  # [win and py2k]
+    # - numpy
+    # - scipy
   run:
     - future
     - python
-    - numpy ==1.11.3 py27hab51878_1207  # [win and py2k]
-    - numpy  # [not (win and py2k)]
-    - scipy ==1.2.1 py27h4c3ab11_0  # [win and py2k]
-    - scipy  # [not (win and py2k)]
+    - {{ pin_compatible('numpy') }}
+    - scipy
     - gmsh <4.0   # [not (win and (py2k or x86))]
```

Comment:
This is a pretty bad constratint to have. You are saying:

* Modern platforms like unix and windows + python3 should be using an old version of gmsh which doesn't seem to have a dedicated branch on conda-forge

Reply:
Our own fipy package does not presently support Gmsh 4.0. I have not been able to provide this support because I've been fighting conda and conda-forge for about 18 months to get a stable set of dependencies that doesn't cause our CIs to fall over every couple of weeks. I have really come to loath everything about this process.
Older Windows either doesn't have a Gmsh 3 build or it doesn't work (I don't remember).

Reply:
RIght, i know packaging can be a pain. Hopefully this package helps. It seems that pytrilinos is kinda old too, causing pinning incompatibilities. I would remove it as a dependency here.